tic-tac-toe.js

```diff
@@ -42,10 +43,13 @@ function toe(board) {
   return ans ? ans : board.every(arr => arr.every(el => el)) ? 0 : -1;
 }
 
-console.log(
-  toe([
-    [0, 0, 0],
-    [1, 1, 1],
-    [2, 1, 2]
-  ])
-);
+let buf;
+process.stdin.on('data', input => {
+  buf = input.toString().split('\n').filter(el => el.length)
+    .map(el => el.trim().split(' ').map(el => parseInt(el)));
+    console.log(buf)
+});
+
+process.stdin.on('end', function() {
+  for (let i = 1; i <= buf[0]; i++) console.log(toe(buf.slice(i, i+3)));
```

Comment:
Have you tried to run the script with two test cases, something like:
```
2
0 0 0
1 1 1
2 1 2
2 2 2
0 1 0
1 0 1
```

Reply:
+

Reply:
My bad. This example:
```
2
0 0 0
1 1 1
2 1 2
0 1 0
1 0 1
2 2 2
```